_posts/zh/newsletters/2022-05-04-newsletter.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Bitcoin Optech Newsletter #198'
+permalink: /zh/newsletters/2022/05/04/
+name: 2022-05-04-newsletter-zh
+slug: 2022-05-04-newsletter-zh
+type: newsletter
+layout: newsletter
+lang: zh
+---
+本周的 Newsletter 总结了一篇关于应用 MuSig2 的文章，转述了一个影响一些旧的 LN 实现的安全问题的披露，讨论了一个通过交易信号来衡量对共识变更的支持的提案，并研究了速率限制对更多带宽效率的闪电网络 gossip 的影响。此外，还包括我们的常规部分，总结新的软件版本和候选版本，以及流行的比特币基础设施项目中值得注意的变更。
+
+## 新闻
+- **MuSig2 的应用说明：** Olaoluwa Osuntokun [回复][osuntokun musig2] 了[Newsletter #195][news195 musig2] 中提到的 [MuSig2][topic musig] 的 BIP 草案，并附上了他和其他人为 btcd 和 LND 所做的实现说明。
+
+  - *与 BIP86 的交互：* 由实施 BIP86 的 [BIP32 HD 钱包][topic bip32]创建的密钥遵循 [BIP341][] 的建议，通过对自身的哈希值进行调整来创建仅有密钥路径的密钥。这有助于防止密钥被用于[多签][topic multisignature]，多签可能会允许一个参与者秘密地包括他们控制的脚本路径支出选项，使他们有能力窃取所有资金。然而，如果多签参与者有意要包括一个脚本路径支出选项，他们需要相互分享他们的密钥的未调整版本。
+
+    Osuntokun建议 BIP86 的实现同时返回原始密钥（内部密钥）和调整后的密钥（输出密钥），以便调用函数可以使用适合其上下文的任何一个。
+
+  - *与脚本路径支出的交互：* 与脚本路径支出一起使用的密钥有一个相关连的问题：为了使用脚本路径，支出者必须知道内部密钥。同样，这就需要实现者返回内部密钥，这样它就可以在其他需要它的代码中使用。
```

Comment:
@harding Our bullet-id-adder function strips out these characters and leaves a blank ID causing build issues. In the past we added some ASCII to get unique IDs and we could do that here if its not possible to keep the Chinese characters in the slugs. Thoughts?

Reply:
I don't get the problem with the slugs, why can't we keep Chinese characters in them? Are only ASCII characters allowed? Could we maybe change some encoding to widen it to UTF-8/16?

Reply:
I found this relevant PR: https://github.com/bitcoinops/bitcoinops.github.io/pull/349#issue-567664307

Reply:
Hi @Xekyo @bitschmidty ! I want to confirm are these 2 operations have equivalent results:
1) `make production` locally
2) PR review checks on GitHub

If so, I will try to fix the problem to make sure there is no error locally and then submit PR again. 
Thanks!

Reply:
And also do you know how to do with this recommendation?
> ... I found that it was easy to fix by just inserting a single-character comment in the bolded text to distinguish the duplicates ...

And since the problems are mostly related to non-Latin text in bullets during generating anchors,  is there any methods that users can assign anchor id manually? 

Thanks very much!

> I found this relevant PR: [#349 (comment)](https://github.com/bitcoinops/bitcoinops.github.io/pull/349#issue-567664307)